packages/cli-hydrogen/src/cli/services/deploy.test.ts

```diff
@@ -93,6 +93,54 @@ describe('deployToOxygen()', () => {
     expect(mockedHealthCheck).toHaveBeenCalledTimes(3)
   })
 
+  it('does retry for createDeployment if the error is recoverable', async () => {
```

Comment:
Could throw these in describe blocks
```suggestion
describe('when error is recoverable')
  it('retry createDeployment')
  
  it('retry uploadDeployment')
```